spec/merchant_spec.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/merchant'
+
+RSpec.describe Merchant do
+  it 'exists' do
+    m = Merchant.new({:id => 5, :name => "Turing School"})
```

Comment:
agree with the space issue. I wasn't aware of the single quote rule when not using interpolation, but it would be a nice way to let people know interpolation was going to happen. I like the curly braces, good fast read indicator that a hash is being passed so I would ignore him there. Would you mind looking up the hash syntax for 1.9 for all of us?

Reply:
This is exactly how I would have done it. I guess there are some Ruby conventions that we haven't been taught yet.

Reply:
So you don't need the curly braces here, and you can swap the lines to read 
```suggestion
id: 5, name: 'Turing School'
```

Reply:
```m = Merchant.new( id: 5, name: "Turing School" )
    m = Merchant.new({:id => 5, :name => "Turing School"})
```